kalliope/core/MainController.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, brain=None):
 
         # create an order listener object. This last will the trigger callback before starting
         self.order_listener = OrderListener(self.analyse_order)
+
+        if self.settings.random_on_ready_answers is not None:
+            Say(message=self.settings.random_on_ready_answers)
+        elif self.settings.random_on_ready_sounds is not None:
+            random_sound_to_play = self._get_random_sound(self.settings.random_on_ready_sounds)
```

Comment:
It seems not done for the random_wake_up_sounds settings, should I add it for both ?
From the code i'm reading now, it means creating a static method in configurationChecker class and call it from settingsLoader when loading one these 2 settings.

Is that the correct way to do it?